File: C#/Extensions/Person.cs
namespace Extensions;

public class Person
{
    public string? Name { get; set; }
    public string? Surname { get; set; }
    public int Age { get; set; }

    public override string ToString()
    {
        return $"Name is:{Name}" +
               $"Surname is: {Surname}" +
               $"Age is: {Age}";
    }
}

File: OOP/Classes3/person.cpp
#include "person.h"

#include <iostream>


void person::eat(char* meal) const
{
	std::cout << "Person eats: " << meal << "\n";
}

void person::print() const
{
	std::cout
	<< this->name << '\n'
	<< this->surname << '\n'
	<< this->age << '\n';
}

